app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -76,20 +75,26 @@ def pr_id
         params[:pr_id]
       end
 
+      def pr_number
+        params[:pr_number]
+      end
+
       def author_reputation_level
         return :major if params[:labels].include?('reputation/contributed_code/major')
         return :minor if params[:labels].include?('reputation/contributed_code/minor')
 
         :regular
       end
 
-      memoize
-      def octokit_client
-        Octokit::Client.new(access_token: github_access_token)
+      def reviews
+        return params[:reviews] if params[:reviews].present?
```

Comment:
This enabled us to short-circuit things a bit. For now, this branch will only be called when syncing pull-request reputation.